### PR TITLE
Added grids and text-utilities

### PR DESCRIPTION
**Pull request by Shrinkhlarajpoot**
Added Grids and text-utilities in madeui

### CODE REVIEW

File: components/GridSimplified/GridSimplified.html [hunk 1]
```diff
@@ -0,0 +1,160 @@
...
+                              </div>
+                        </div>
+                    </div>
+                    <iframe
+                    src="https://carbon.now.sh/embed/J6yqOsfhCfLydlOeQ4x7"
+                    style="width:90%; height:150px; border:0; transform: scale(1); overflow:hidden;"
+                    sandbox="allow-scripts allow-same-origin">
+                  </iframe>
+                 <h3 class="sub_heading">Example of side by side (30:70) layout</h3>
+                  <div>Add grid <span style="color:var(--primary-color)">grid 30-70 layout </span>class on wrapper div which is wrapping 2 Divs</div>
```
Comment (msdeshmukh009) [inline components/GridSimplified/GridSimplified.html:109]: Maybe try avoiding inline style.

File: components/Text-Utlities/Text-Utilities.html [hunk 2]
```diff
@@ -0,0 +1,160 @@
...
+                 style="width:90%; height:150px; border:0; transform: scale(1); overflow:hidden;"
```
Comment (msdeshmukh009) [inline components/Text-Utlities/Text-Utilities.html:118]: You can also make class for this style as well.

**Review by msdeshmukh009 — APPROVED** [review]
The code looks fine to me. I think you have not updated the documentation page as I can't find the link for the grid and text utility page.

File: components/GridSimplified/GridSimplified.css [hunk 3]
```diff
@@ -0,0 +1,41 @@
...
+
+.grid-30-70-layout{
+    width:80%;
+    height:fit-content;
+    display: grid;
+    grid-template-columns: 30% 70%;
+    border:1px solid var(--error--color);
+   
+}
+.grid-four-layout{
```
Comment (Chakravarthi0) [inline components/GridSimplified/GridSimplified.css:32]: Hey shrinkhla,

please correct me if I am wrong. We are supposed to make grid-2 and grid-3. You can definitely have additional components like grid-4. But grid-3 is a must, I guess.

File: components/Text-Utlities/Text-Utilities.css [hunk 4]
```diff
@@ -0,0 +1,58 @@
+.typo_container{
+    width:90%;
+    height:fit-content;
+    background:rgb(229,236,238);
```
Comment (Chakravarthi0) [inline components/Text-Utlities/Text-Utilities.css:4]: you could use a variable for background.

**Review by Chakravarthi0 — CHANGES_REQUESTED** [review]

File: components/GridSimplified/GridSimplified.css (hunk 3 above)
Comment (Shrinkhlarajpoot) [reply components/GridSimplified/GridSimplified.css:32]: Yeah,its important as told to us..Thanks for reminding me.

File: components/Text-Utlities/Text-Utilities.html (hunk 2 above)
Comment (Shrinkhlarajpoot) [reply components/Text-Utlities/Text-Utilities.html:118]: okay

**Pull request merged**